From 4443a91ec04f41a0f37b2e355bbd5cdf545dcd98 Mon Sep 17 00:00:00 2001
From: Tomer Maimon <tmaimon77@gmail.com>
Date: Thu, 4 Jul 2024 14:32:16 +0300
Subject: [PATCH 17/23] arm64: dts: npcm8xx: add clock properties to reset node

Add clock-cell and reference clock to reset node since the reset driver
is register the NPCM8xx clock controller aux device.

Signed-off-by: Tomer Maimon <tmaimon77@gmail.com>
---
 arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi b/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
index 72828d91b33a..ce6fb94bd491 100644
--- a/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
+++ b/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
@@ -76,6 +76,8 @@ rstc: reset-controller@f0801000 {
 			reg = <0x0 0xf0801000 0x0 0x78>;
 			#reset-cells = <2>;
 			nuvoton,sysgcr = <&gcr>;
+			clocks = <&refclk>;
+			#clock-cells = <1>;
 		};
 
 		clk: clock-controller@f0801000 {
-- 
2.43.0

